www/docs/en/dev/guide/platforms/ios/index.md

```diff
@@ -74,190 +67,123 @@ There are two ways to download Xcode:
 
-3. Select the intended device from the toolbar's __Scheme__ menu, such
-   as the iPhone 6.0 Simulator as highlighted here:
+### Using build.json
 
-   ![]({{ site.baseurl }}/static/img/guide/platforms/ios/select_xcode_scheme.png)
+Alternatively, you could specify them in a build configuration file (`build.json`)
+using the `--buildConfig` argument to the same commands. Here's a sample of a
+build configuration file:
 
-4. Press the __Run__ button that appears in the same toolbar to the
-   left of the __Scheme__. That builds, deploys and runs the
-   application in the emulator. A separate emulator application opens
-   to display the app:
+    {
+         "ios": {
+             "debug": {
+                 "codeSignIdentitiy": "iPhone Development",
+                 "provisioningProfile": "926c2bd6-8de9-4c2f-8407-1016d2d12954",
+             },
+             "release": {
+                 "codeSignIdentitiy": "iPhone Distribution"
+                 "provisioningProfile": "70f699ad-faf1-4adE-8fea-9d84738fb306",
+             }
+         }
+    }
+    
+### Using xcrun
 
-   ![]({{ site.baseurl }}/static/img/guide/platforms/ios/HelloWorldStandard.png)
+You can also sign from the command line using the following command:
 
-   Only one emulator may run at a time, so if you want to test the app
-   in a different emulator, you need to quit the emulator application
-   and run a different target within Xcode.
+```
+    xcrun -sdk iphoneos PackageApplication -v /home/user/app/build/device/MyApp.app -o /home/user/app/build/device/MyApp.ipa --sign "iPhone Development" --embed "7151ab45-6085-4ea1-9bcd-022b5cebe44b"
+```
 
-Xcode comes bundled with emulators for the latest versions of iPhone
-and iPad. Older versions may be available from the __Xcode &rarr;
-Preferences &rarr; Downloads &rarr; Components__ panel.
+## Debugging
 
-## Deploy to Device
+For details on the debugging tools that come with Xcode, see this [article](https://developer.apple.com/support/debugging)
+and this [video](https://developer.apple.com/videos/play/wwdc2014-413/).
 
-For details about various requirements to deploy to a device, refer
-to the _Launch Your App On Devices_ section of
-Apple's
-[About App Distribution Workflows](https://developer.apple.com/library/prerelease/ios/documentation/IDEs/Conceptual/AppDistributionGuide/Introduction/Introduction.html).
-Briefly, you need to do the following before deploying:
+### Open a Project within Xcode
 
-1. Join the Apple iOS Developer Program.
+Cordova for iOS projects can be opened in Xcode. This can be useful if 
+you wish to use Xcode built in debugging/profiling tools or if you are
+developing iOS plugins. Please note that when opening your project in Xcode, 
+it is recommended that you do NOT edit your code in the IDE. This will edit the code 
```

Comment:
You can edit the root www folder from Xcode. But doing this makes o effect when you run the app. You have to do a 'cordova prepare ios' to copy the changes. I use a xcode build script that run 'cordova prepare ios' before running. The files you should change are the ones on "Staging" folder, those are the  ```platforms``` ones